src/content/docs/extensions/llm.mdx

```diff
@@ -0,0 +1,232 @@
+---
+title: "LLM Extension"
+---
+
+The **LLM extension** provides an interface to fetch text embeddings from supported providers and their models.
+
+It can be installed and loaded using the following commands:
+
+```sql
+INSTALL llm;
+LOAD llm;
+```
+
+See the list of providers and their embedding documentation.
+
+* [Open-AI](https://platform.openai.com/docs/guides/embeddings)
+* [Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/titan-embedding-models.html)
+* [Google Vertex](https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/get-text-embeddings)
+* [Google Gemini](https://ai.google.dev/gemini-api/docs/embeddings)
+* [Voyage AI](https://docs.voyageai.com/docs/embeddings)
+* [Ollama](https://ollama.com/blog/embedding-models)
+
+:::note[Note]
+Model parameters are dynamically validated at runtime. If an unsupported model is specified, the provider's API will return an appropriate error.
```

Comment:
Move this to where the syntax is described, because that is where the user is thinking about parameters